ballerina/Ballerina.toml

```diff
@@ -12,3 +12,11 @@ path = "./lib/sql-native-@stdlib.sql.native.version@.jar"
 [[platform.java11.dependency]]
 path = "./lib/ojdbc8-@oracledb.driver.version@.jar"
 scope = "testOnly"
+
+[[platform.java11.dependency]]
+path = "./lib/xdb-@oracle.xdb.version@.jar"
+scope = "testOnly"
+
+[[platform.java11.dependency]]
+path = "./lib/xmlparserv2-@oracle.xmlparserv2.version@.jar"
+scope = "testOnly"
```

Comment:
If the user needs to use XML types with the oracle connector, does he need to add these external libs to the project? what if he not added and try to use XML types?

Reply:
Are these jar versions, directly binds with the driver jar version

Reply:
official documentation for jars : https://www.oracle.com/database/technologies/maven-central-guide.html

these jars are affecting in runtime. so if user does not add those jars, an ApplicationError is returned to the user.

Reply:
what is the error returned to the user?

Reply:
if both xdb.jar and xmlparserv2.jar  not present then 

```
error ApplicationError ("Error when iterating through the SQL result. oracle/xdb/XMLType")
```

if xdb.jar present and xmlparserv2.jar is not present then 

```
 java.lang.NoClassDefFoundError {"message":"oracle/xml/parser/v2/XMLParseException","cause":error("java.lang.ClassNotFoundException",message="oracle.xml.parser.v2.XMLParseException")}
```

Reply:
Error messages are not correct. we need meaningful messages so that users know what is missing. Shall we also check the licenses and check whether we can pack with our connector

Reply:
error messages updated in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/10/commits/6fff1c2346a142ead9380249379a1088ff13ee71